src/Lib/CakeboxExecute.php

```diff
@@ -161,36 +161,43 @@ public function gitConfig($gitKey, $value)
     }
 
     /**
-     * Self-update cakebox console and dashboard by updating git repository and
+     * Self-update cakebox-console projeect by self-updating Composer (to
+     * prevent outdated warning), updating the git repository and finally
      * running composer update.
      *
      * @return boolean True on success
      */
     public function selfUpdate()
     {
-        log::debug("Self-updating cakebox-console...");
+        log::debug('Self-updating Composer...');
```

Comment:
Please use Log instead of log, i thoght i had replaced them all already, seems i missed some